[packit] Propose downstream failed for release 353

Packit failed on creating pull-requests in dist-git (https://src.fedoraproject.org/rpms/cockpit.git):

<table><tr><th>dist-git branch</th><th>error</th></tr><tr><td><code>f42</code></td><td>See <a href="https://dashboard.packit.dev/jobs/propose-downstream/23076">https://dashboard.packit.dev/jobs/propose-downstream/23076</a></td></tr>
</table>


You can retrigger the update by adding a comment (`/packit propose-downstream`) into this issue.

I tried to reproduce this locally but can't but this only occurs on Fedora 42:
```
Submodule path 'vendor/ferny': checked out 'c8713c4d3ba74a9d8db1ce8e0a15296e6a333257'
Submodule path 'vendor/systemd_ctypes': checked out 'c7df9c0114641e2063dc56608c4671fe73a984fb'
  GEN      cockpit-353.dev0+gd855d9f10/src/cockpit/_version.py
configure.ac:18: warning: file 'version.m4' included several times
make[3]: *** [Makefile:5959: ../../dist/static/manifest.json] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [Makefile:5170: distdir-am] Error 2
make[1]: *** [Makefile:5802: distdir] Error 2
make: *** [Makefile:5247: dist] Error 2
+ tar -xJf '' --wildcards '*/tools/cockpit.spec' --strip-components=2
tar (child): : Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Fedora rawhide [hangs without any output](https://dashboard.packit.dev/jobs/propose-downstream/23066).
Fedora 43, task was accepted but nothing in the queue.

For the rawhide/43 issues we should reach out to the packit team.

--- a/Makefile.am
+++ b/Makefile.am
@@ -63,10 +63,10 @@ $(distdir)/configure: $(distdir)/version.m4
 
 # Various downstream packaging assets
 
-NODE_CACHE = cockpit-node-$(VERSION).tar.xz
+NODE_CACHE = $(CURDIR)/cockpit-node-$(VERSION).tar.xz
 
 $(NODE_CACHE): $(srcdir)/package-lock.json
-	$(AM_V_GEN) $(srcdir)/tools/node-modules runtime-tar $(CURDIR)/$(NODE_CACHE)
+	$(AM_V_GEN) $(srcdir)/tools/node-modules runtime-tar $(NODE_CACHE)
 
 dist-hook: $(distdir)/tools/debian/copyright
 # when building from a git checkout, we need to generate the copyright file

